### ISSUE TITLE
is it possible to use this library on device?

### ISSUE THREAD

**Issue by Andriiklymiuk** [destinationstransfers/passkit#227]
Hi,

We try to use PassKit directly in our react native project. Is it somehow possible to use it that way and generate pass on device? Because now we are getting 'there is no module http2', etc. errors. 

And by the way, thanks for the library)

**Comment by bukira** [comment]
I think you need a server to sign passes

**Comment by Andriiklymiuk** [comment]
oh, ok, thanks)

**Comment by tinovyatkin** [comment]
@bukira  no, you don't need anything apart of this library to create pass  - it's pure JavaScript implementation. On native script you probably will need to save buffer to a file then open it.

**Comment by guisehn** [comment]
Even if could be possible to generate the pass using React Native directly in the device using the library, I don't think you should do that because you'd need to store your Apple Developer certificate and private key inside your app code. If someone is able to reverse engineer and get access to your code, they could issue passes in your behalf. That's why having a secure web server to issue your passes would be a better option.

**Comment by guisehn** [comment]
Even though it could be possible to generate the pass using React Native directly in the device using the library, I don't think you should do that because you'd need to store your Apple Developer certificate and private key inside your app code. If someone is able to reverse engineer and get access to your code, they could issue passes on your behalf. That's why having a secure web server to issue your passes would be a better option.
